dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/logging/CXFLogger.java

```diff
@@ -0,0 +1,180 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.apache.cxf.common.logging;
+
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.websphere.ras.TraceStateChangeListener;
+import com.ibm.websphere.ras.annotation.Trivial;
+import com.ibm.ws.logging.internal.StackFinder;
+
+
+/**
+ *  CXFLogger is aimed to add class name and method name if provided to logs. 
+ *  The scope of this logger is is limited to CXF bundles.
+ */
+public class CXFLogger extends Logger implements TraceStateChangeListener {
```

Comment:
There are many other log methods that aren't being overridden, is there a reason you are only overriding these handful?